src/app/projects/components/project-settings/project-settings.component.html

```diff
@@ -89,6 +89,9 @@
         [useableAttributes]="useableAttributes" [attributes]="attributes" [lh]="lh">
     </kern-labeling-tasks>
 
+    <kern-gates-integrator [project]="project">
```

Comment:
condition for only on manged missing

Reply:
*ngIf="isManaged" should do it however isManaged should be defaulted to false not true (otherwise it would blink in and out after the info is collected)

Reply:
changed